db/migrate/20250807154439_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,57 @@
+# This migration comes from active_storage (originally 20170806125915)
```

Comment:
Was this generated or copied from another codebase? If the latter, what would you think about leaning on Rails tooling to generate this anew? ActiveStorage has changed a bunch over the years.

```shell
bin/rails active_storage:install
bin/rails db:migrate
```

See https://guides.rubyonrails.org/active_storage_overview.html#setup